src/cgame/cg_rocket_datasource.cpp

```diff
@@ -1647,6 +1647,12 @@ void CG_Rocket_BuildAlienEvolveList( const char *table )
 		{
 			if ( BG_Class( i )->team == TEAM_ALIENS )
 			{
+				if ( ( i == PCL_ALIEN_BUILDER0 && ( BG_ClassUnlocked( PCL_ALIEN_BUILDER0_UPG ) && !BG_ClassDisabled( PCL_ALIEN_BUILDER0_UPG ) ) )||
```

Comment:
I believe that we should not include this change in the "backend". This is a strictly UI change that should be made in the UI...The circleMenu can use a filter function or something that can filter things out like this...

Reply:
Is CG_Rocket_DFCMAlienEvolve the right place for it?

Reply:
Yeah, it's fine to put it there.